html-css/week3/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./styles.css" />
+    <title>Info</title>
+</head>
+
+<body>
+    <form action="./index2.html" method="GET">
+        <div class="navbar">
+            <img src="" alt="logo">
+            <nav>
+                <a href="blank">Menu</a>
+                <a href="blank">Contacts</a>
+                <a href="blank">About Us</a>
+            </nav>
```

Comment:
When you're creating navigation bar, try to wrap your  `a` tag with ` ul li` and put a bit `gap` between them, tabs are too close to each other